defaults/main.yml

```diff
@@ -13,5 +13,17 @@ letsencrypt_certbot_verbose: true
 letsencrypt_pause_services: []
 # Force Certificate Reneval
 letsencrypt_force_renew: true
-# Additional certbot arguments
-letsencrypt_certbot_args: []
+# certbot required parameters
+letsencrypt_certbot_default_args:
+  - "certonly"
```

Comment:
No strong preference, it's just that they're not required.

   items:
     - "foo"

   items:
     - "foo"

   items: ["foo"]

As said, for a list of cli params, I think the last one is clearer. But for other YAML lists, I prefer the first one. It's probably better stay consistent to one or the other in this role. Since the first one is the more common one in Ansible roles afaik, I'd say lets go with that one, without the quotes if possible.